src/Features/Core/Portable/Shared/Extensions/DocumentExtensions.cs

```diff
@@ -154,5 +155,21 @@ public static async Task<NamingRule> GetApplicableNamingRuleAsync(this Document
 
             throw ExceptionUtilities.Unreachable;
         }
+
+        public static ImmutableArray<AbstractFormattingRule> GetFormattingRules(this Document document, IEnumerable<AbstractFormattingRule>? additionalRules, TextSpan span)
```

Comment:
is this called from multiple places?

Reply:
argument order is a bit odd.  i would expect it to be the required 'TextSpan' first, then the optional additionalRules next.